### PR TITLE
enable melodic checks

### PR DESCRIPTION
**Pull request by jabbenseth**

### CODE REVIEW

**Comment by jabbenseth** [comment]
works towards https://github.com/ipa320/care-o-bot/issues/52

**Review by pschnattinger — DISMISSED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -13,10 +13,12 @@ env:
...
     - ROS_DISTRO=kinetic UPSTREAM_WORKSPACE=file
     - ROS_DISTRO=kinetic UPSTREAM_WORKSPACE=debian
+    - ROS_DISTRO=melodic UPSTREAM_WORKSPACE=file
+    - ROS_DISTRO=melodic UPSTREAM_WORKSPACE=debian
 matrix:
   allow_failures:
-    - env: ROS_DISTRO=indigo UPSTREAM_WORKSPACE=debian
-    - env: ROS_DISTRO=kinetic UPSTREAM_WORKSPACE=debian
+    - env: ROS_DISTRO=melodic UPSTREAM_WORKSPACE=file
+    - env: ROS_DISTRO=melodic UPSTREAM_WORKSPACE=debian
```
Comment (fmessmer) [inline .travis.yml]: as all six travis jobs succeed, I would remove the `allow_failures` block completely

**Review by fmessmer — CHANGES_REQUESTED** [review]
should I release this repo into `melodic`?
I'm about to do another tagging...and release session for some of the cob packages anyway
@ipa-jba

**Comment by jabbenseth** [comment]
a release for melodic would be great. I will remove the allow failure tag. Kept it since it is part of the accepted template in cob_common

**Comment by fmessmer** [comment]
@ipa-jba 
as `cob_common` is already released into `melodic` (http://repositories.ros.org/status_page/ros_melodic_default.html?q=cob), we might want to remove the `allow_failures` there, too - could you provide a PR there, too?

the `allow_failures` were for testing whether the tests **would** pass for `melodic` :wink:

**Review by fmessmer — APPROVED** [review]

**Comment by jabbenseth** [comment]
https://github.com/ipa320/cob_common/pull/267 there you go

**Pull request merged**